includes/class-comment.php

```diff
@@ -335,4 +346,29 @@ public static function generate_id( $comment ) {
 			\trailingslashit( site_url() )
 		);
 	}
+
+	/**
+	 * Enqueue scripts for remote comments
+	 */
+	public static function enqueue_scripts() {
+		$handle     = 'activitypub-remote-reply-script';
+		$data       = array(
+			'namespace' => ACTIVITYPUB_REST_NAMESPACE,
+		);
+		$js         = sprintf( 'var _activityPubOptions = %s;', wp_json_encode( $data ) );
+		$asset_file = ACTIVITYPUB_PLUGIN_DIR . 'build/remote-reply/index.asset.php';
+
+		if ( \file_exists( $asset_file ) ) {
+			$assets = require_once $asset_file;
+
+			\wp_enqueue_script(
+				$handle,
+				\plugins_url( 'build/remote-reply/index.js', __DIR__ ),
+				$assets['dependencies'],
+				'1.0',
+				true
+			);
+			\wp_add_inline_script( $handle, $js, 'before' );
```

Comment:
Need to add
```php
wp_enqueue_style( 'wp-components' );
```

For the Dialog to appear properly.

Reply:
Thanks 🫣